routes/api.js

```diff
@@ -160,6 +161,124 @@ router.ws('/results', (ws, req, next) => {
 
 router.ws('/conversation', conversationHandler);
 
+router.post('/conversation/startRecording', (req, res, next) => {
```

Comment:
This is the only camelCased endpoint we have, so, weird? Also, the dual of start should be stop, not end?